runtime/common/src/weights/extrinsic_weights.rs

```diff
@@ -43,17 +43,17 @@ parameter_types! {
 	/// Calculated by multiplying the *Average* with `1.0` and adding `0`.
 	///
 	/// Stats nanoseconds:
-	///   Min, Max: 98_988, 105_759
-	///   Average:  100_451
-	///   Median:   100_304
-	///   Std-Dev:  1082.34
+	///   Min, Max: 100_113, 110_777
+	///   Average:  101_295
+	///   Median:   101_040
+	///   Std-Dev:  1335.33
```

Comment:
Just noting that this is a jump of ~30% in std-dev which means the weight calculations are varying more widely in this benchmark run.